src/acl.c

```diff
@@ -413,6 +413,38 @@ void ACLSetUserCommandBit(user *u, unsigned long id, int value) {
     }
 }
 
+void ACLChangeCommandPerm(user *u, struct redisCommand *cmd, int allow) {
```

Comment:
This function could use some documentation, I didn't really follow what it did until looking at in the context of other places it's called.